tests/test_parallel_assert.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from pytest_mpi.parallel_assert import parallel_assert
+
+
+@pytest.mark.parametrize('expression', [True, False])
+def test_parallel_assert_equivalent_to_assert_in_serial(expression):
+    raised_exception = True
+
+    try:
+        parallel_assert(expression)
+        raised_exception = False
+    except AssertionError:
+        try:
+            assert expression
+        except AssertionError:
+            pass
+
+    if not raised_exception:
+        assert expression
```

Comment:
I think below is clearer:
```suggestion
    try:
        parallel_assert(expression)
        parallel_raised_exception = False
    except AssertionError:
        parallel_raised_exception = True  
    try:
        assert expression
        serial_raised_exception = False
    except AssertionError:
        serial_raised_exception = True

    assert serial_raised_exception == parallel_raised_exception
```